Modules/Microsoft365DSC/Examples/Resources/AADIdentityGovernanceLifecycleWorkflow/2-Update.ps1

```diff
@@ -0,0 +1,67 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        AADIdentityGovernanceLifecycleWorkflow "AADIdentityGovernanceLifecycleWorkflow-Onboard pre-hire employee updated version"
+        {
+            Category             = "joiner";
+            Credential           = $Credscredential;
```

Comment:
We should use service principal (AppId, TenantId, CertThumprint) for the example. CredsCredential isn't defined anyway.